src/Backend/Modules/Pages/Installer/Installer.php

```diff
@@ -65,14 +65,147 @@ private function installExampleData(): void
+        $extras['blog_widget_recent_comments'] = $this->insertExtra(
+            'Blog',
+            ModuleExtraType::widget(),
+            'RecentComments',
+            'RecentComments'
+        );
+        $extras['blog_widget_categories'] = $this->insertExtra(
+            'Blog',
+            ModuleExtraType::widget(),
+            'Categories',
+            'Categories'
+        );
         $extras['blog_widget_archive'] = $this->insertExtra('Blog', ModuleExtraType::widget(), 'Archive', 'Archive');
-        $extras['blog_widget_recent_articles_full'] = $this->insertExtra('Blog', ModuleExtraType::widget(), 'RecentArticlesFull', 'RecentArticlesFull');
-        $extras['blog_widget_recent_articles_list'] = $this->insertExtra('Blog', ModuleExtraType::widget(), 'RecentArticlesList', 'RecentArticlesList');
+        $extras['blog_widget_recent_articles_full'] = $this->insertExtra(
+            'Blog',
+            ModuleExtraType::widget(),
+            'RecentArticlesFull',
+            'RecentArticlesFull'
+        );
+        $extras['blog_widget_recent_articles_list'] = $this->insertExtra(
+            'Blog',
+            ModuleExtraType::widget(),
+            'RecentArticlesList',
+            'RecentArticlesList'
+        );
+        $extras['faq_block'] = $this->insertExtra('Faq', ModuleExtraType::block(), 'Faq');
+        $extras['mailmotor_subscribe'] = $this->insertExtra(
+            'Mailmotor',
+            ModuleExtraType::block(),
+            'SubscribeForm',
+            'Subscribe'
+        );
+        $extras['mailmotor_unsubscribe'] = $this->insertExtra(
+            'Mailmotor',
+            ModuleExtraType::block(),
+            'UnsubscribeForm',
+            'Unsubscribe'
+        );
+        $extras['tags_block'] = $this->insertExtra('Tags', ModuleExtraType::block(), 'Tags');
+        $extras['profiles_activate'] = $this->insertExtra(
+            'Profiles',
+            ModuleExtraType::block(),
+            'Activate',
+            'Activate'
+        );
+        $extras['profiles_forgot_password'] = $this->insertExtra(
+            'Profiles',
+            ModuleExtraType::block(),
+            'ForgotPassword',
+            'ForgotPassword'
+        );
+        $extras['profiles_block'] = $this->insertExtra(
+            'Profiles',
+            ModuleExtraType::block(),
+            'Dashboard'
+        );
+        $extras['profiles_login'] = $this->insertExtra(
+            'Profiles',
+            ModuleExtraType::block(),
+            'Login',
+            'Login'
+        );
+        $extras['profiles_logout'] = $this->insertExtra(
+            'Profiles',
+            ModuleExtraType::block(),
+            'Logout',
+            'Logout'
+        );
+        $extras['profiles_change_email'] = $this->insertExtra(
+            'Profiles',
+            ModuleExtraType::block(),
+            'ChangeEmail',
+            'ChangeEmail'
+        );
+        $extras['profiles_change_password'] = $this->insertExtra(
```

Comment:
don't know that by heart, you'll have to look for it, or add it if needed.
But wouldn't it be better to just add these things in the installers of the modules?